tools/db_crashtest.py

```diff
@@ -1080,8 +1074,6 @@ def whitebox_crash_main(args, unknown_args):
             print("TEST FAILED. See kill option and exit code above!!!\n")
             sys.exit(1)
 
-        #stderr already printed above
-        exit_if_stderr_has_errors(stderrdata, False)
```

Comment:
we already print stderror above and this statement is now redundant. So removing this line too